python/pyspark/sql/connect/merge.py

```diff
@@ -0,0 +1,257 @@
+#
```

Comment:
Can you add this file in `dev/sparktestsupport/modules.py` so it can run doctests? For Spark Classic one too

Reply:
Done!